tensorflow/workspace2.bzl

```diff
@@ -126,20 +126,21 @@ def _tf_toolchains():
 def _tf_repositories():
     """All external dependencies for TF builds."""
 
-    # To update any of the dependencies bellow:
+    # To update any of the dependencies below:
     # a) update URL and strip_prefix to the new git commit hash
     # b) get the sha256 hash of the commit by running:
     #    curl -L <url> | sha256sum
     # and update the sha256 with the result.
 
-    # LINT.IfChange
+    # Attention: TensorFlow Lite CMake build uses this variable, update only the hash content. 
```

Comment:
This comment looks confusing since when we update commit hash, we need to update sha256 and prefix.